app/assets/javascripts/admin/addon/components/admin-report-table-cell.js

```diff
@@ -1,21 +1,32 @@
-import Component from "@ember/component";
+import classic from "ember-classic-decorator";
+import {
+  attributeBindings,
+  classNameBindings,
+  classNames,
+  tagName,
+} from "@ember-decorators/component";
 import { alias } from "@ember/object/computed";
+import Component from "@ember/component";
 import discourseComputed from "discourse-common/utils/decorators";
 
-export default Component.extend({
-  tagName: "td",
-  classNames: ["admin-report-table-cell"],
-  classNameBindings: ["type", "property"],
-  attributeBindings: ["value:title"],
-  options: null,
+@classic
+@tagName("td")
+@classNames("admin-report-table-cell")
+@classNameBindings("type", "property")
+@attributeBindings("value:title")
+export default class AdminReportTableCell extends Component {
+  options = null;
 
+  @alias("label.type")
+  type;
+  @alias("label.mainProperty")
+  property;
+  @alias("computedLabel.formattedValue")
+  formattedValue;
+  @alias("computedLabel.value")
+  value;
```

Comment:
How do we feel about this two-line format? 😃

Reply:
🤢